src/ApplicationCore/Exceptions/FraudProtectionApiException.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Net.Http;
+
+namespace Contoso.FraudProtection.ApplicationCore.Exceptions
+{
+    public class FraudProtectionApiException : Exception
+    {
+        public HttpResponseMessage Response { get; }
+
+        public FraudProtectionApiException()
+        {
+        }
+
+        public FraudProtectionApiException(HttpResponseMessage response) : base($"Fraud Protection API exception occurred!")
+        {
+            Response = response;
+        }
```

Comment:
it looks you have just used this one in below code. Do you need all other methods, as most of the sample app code is written by you may be you need more than anyone.

Reply:
These are just the standard Exception constructors.  I agree either way, but don't mind this extra code to match the typical exception constructors so I'll leave it.  The other custom exception we have in this same folder has these constructors as well.